docs/api-reference/3d-tiles/tile-3d-header.md

```diff
@@ -103,28 +105,26 @@ Get the tile's screen space error.
 
 Update the tile's visibility.
 
-### requestContent()
+### loadContent()
 
 Requests the tile's content.
 
-The request may not be made if the Request Scheduler can't prioritize it.
-
 ### unloadContent()
 
 Unloads the tile's content.
 
-### visibility(frameState, parentVisibilityPlaneMask)
+### visibility(frameState : FrameState, parentVisibilityPlaneMask : Number)
```

Comment:
where is the definition of FrameState?

Reply:
> where is the definition of FrameState?

Yes this is the most important part currently left out, will add a task to document that.